src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books';
+import filterReducer from './filter';
```

Comment:
- [ ] according to the requirements in this component you should Import `createStore`, `combineReducers`  and `applyMiddleware`  from `redux`